Updated Readme.md to include new org under help wanted label

rohansen856 opened the pull request:
**Description**

This PR fixes #6393

**Notes for Reviewers**


**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

vishalvivekm commented:
See https://github.com/meshery/meshery/issues/14549

l5io commented:
🚀 Preview for commit ae3fc7478b20f692ac9625f9a6fa43040b6cb012 at: https://680f749e6527a2936fb75d0f--layer5.netlify.app

rohansen856 commented:
@vishalvivekm I have updated the link to properly filter the issues by orgs. attaching a screenshot for reference:
![image](https://github.com/user-attachments/assets/04bbaadd-60c9-42ec-a3b4-ccd83063dda5)

corrected url for reference: [https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+(org%3Alayer5io+OR+org%3Ameshery+OR+org%3Alayer5labs+OR+org%3Aservice-mesh-performance+OR+org%3Aservice-mesh-patterns+OR+org%3Ameshery-extensions)+label%3A%22help+wanted%22](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+(org%3Alayer5io+OR+org%3Ameshery+OR+org%3Alayer5labs+OR+org%3Aservice-mesh-performance+OR+org%3Aservice-mesh-patterns+OR+org%3Ameshery-extensions)+label%3A%22help+wanted%22)

the changes are in accordance to what is mentioned in [here](https://github.com/meshery/meshery/issues/14549) and working properly (although the invalid organization message is still there, but the filters do work).
Thanks.

l5io commented:
🚀 Preview for commit 5b57c57b023d5e423e82de2ad9b2e130827253e0 at: https://680f81d850e6131d82245e24--layer5.netlify.app

sudhanshutech commented:
@vishalvivekm lgtm
